sources/packages/backend/apps/queue-consumers/src/main.ts

```diff
@@ -29,10 +29,17 @@ import { SystemUsersService } from "@sims/services";
   // Create bull board UI dashboard for queue management.
   const serverAdapter = new ExpressAdapter();
   serverAdapter.setBasePath("/admin/queues");
-  const bullBoardQueues = queues.map((queue) => {
-    return new BullAdapter(app.get<Queue>(`BullQueue_${queue.name}`), {
-      readOnlyMode: queue.dashboardReadonly,
-    });
+  const bullBoardQueues: BullAdapter[] = [];
+  queues.forEach((queue) => {
+    if (!queue.isActive && queue.cron) {
```

Comment:
I believe the model can return a property that would indicate if it is a scheduler.
```ts
isScheduler: !!queue.queueConfiguration.cron,
```